infrastructure_files/getting-started-with-zitadel.sh

```diff
@@ -402,29 +402,126 @@ read_nb_domain() {
   echo "$READ_NETBIRD_DOMAIN"
 }
 
+check_nb_port() {
+  HTTPS_PORT=$1
+  if [ "$HTTPS_PORT-x" == "-x" ]; then
+    echo "The NETBIRD_PORT variable cannot be empty." > /dev/stderr
+    return 1
+  fi
+  return 0
+}
+
+read_nb_port() {
+  READ_NETBIRD_PORT=""
+  echo -n "Enter the https port you want to use for NetBird (e.g. 443): " > /dev/stderr
+  read -r READ_NETBIRD_PORT < /dev/tty
+  if ! check_nb_port "$READ_NETBIRD_PORT"; then
+    read_nb_port
+  fi
+  echo "$READ_NETBIRD_PORT"
+}
+
+
+check_nb_http_port() {
+  HTTP_PORT=$1
+  if [ "$HTTP_PORT-x" == "-x" ]; then
+    echo "The NETBIRD_HTTP_PORT variable cannot be empty." > /dev/stderr
+    return 1
+  fi
+  return 0
+}
+
+read_nb_http_port() {
+  READ_NETBIRD_HTTP_PORT=""
+  echo -n "Enter the http port you want to use for NetBird (e.g. 80): " > /dev/stderr
+  read -r READ_NETBIRD_HTTP_PORT < /dev/tty
+  if ! check_nb_http_port "$READ_NETBIRD_HTTP_PORT"; then
+    read_nb_http_port
+  fi
+  echo "$READ_NETBIRD_HTTP_PORT"
+}
+
+check_nb_8080_port() {
+  PORT_8080=$1
+  if [ "$PORT_8080-x" == "-x" ]; then
+    echo "The NETBIRD_8080_PORT variable cannot be empty." > /dev/stderr
+    return 1
+  fi
+  return 0
+}
+
+read_nb_8080_port() {
+  READ_NETBIRD_8080_PORT=""
+  echo -n "Enter the NETBIRD_8080_PORT you want to use for NetBird (e.g. 8080): " > /dev/stderr
+  read -r READ_NETBIRD_8080_PORT < /dev/tty
+  if ! check_nb_8080_port "$READ_NETBIRD_8080_PORT"; then
+    read_nb_8080_port
+  fi
+  echo "$READ_NETBIRD_8080_PORT"
+}
+
+
+check_nb_3478_port() {
+  PORT_3478=$1
+  if [ "$PORT_3478-x" == "-x" ]; then
+    echo "The NETBIRD_3478_PORT variable cannot be empty." > /dev/stderr
+    return 1
+  fi
+  return 0
+}
+
+read_nb_3478_port() {
+  READ_NETBIRD_3478_PORT=""
+  echo -n "Enter the TURN_LISTENING_PORT you want to use for NetBird (e.g. 3478): " > /dev/stderr
+  read -r READ_NETBIRD_3478_PORT < /dev/tty
+  if ! check_nb_3478_port "$READ_NETBIRD_3478_PORT"; then
+    read_nb_3478_port
+  fi
+  echo "$READ_NETBIRD_3478_PORT"
+}
+
 initEnvironment() {
   CADDY_SECURE_DOMAIN=""
   ZITADEL_EXTERNALSECURE="false"
   ZITADEL_TLS_MODE="disabled"
   ZITADEL_MASTERKEY="$(openssl rand -base64 32 | head -c 32)"
-  NETBIRD_PORT=80
   NETBIRD_HTTP_PROTOCOL="http"
   TURN_USER="self"
   TURN_PASSWORD=$(openssl rand -base64 32 | sed 's/=//g')
   TURN_MIN_PORT=49152
   TURN_MAX_PORT=65535
 
```

Comment:
Let's set default ports to be used when the user is not using the interactive mode.